Source/JavaScriptCore/llint/InPlaceInterpreter.asm

```diff
@@ -384,6 +384,92 @@ macro ipintException(exception)
     jmp _wasm_throw_from_slow_path_trampoline
 end
 
+# OSR
+
+macro ipintPrologueOSR(increment)
+    loadp WasmCodeBlock[cfr], ws0
+    baddis increment, Wasm::IPIntCallee::m_tierUpCounter + Wasm::LLIntTierUpCounter::m_counter[ws0], .continue
+
+if ARM64 or ARM64E
+    push a0, a1, a2, a3, a4, a5, a6, a7
+    subq 32, sp
+    storepaird fa0, fa1, [sp]
+    storepaird fa2, fa3, 16[sp]
```

Comment:
Is `fa3` enough? I think wasm ARM64 has upto wfa7.
It would be nice if we can use `forEachArgumentFPR` here.